archspec/cli.py

```diff
@@ -13,11 +13,13 @@
 
 
 @click.group(name="archspec")
+@click.version_option()
 def main():
     """archspec command line interface"""
 
 
 @main.command()
+@click.version_option()
```

Comment:
Why does cpu need a version option? Seems like that can just be on the root command.

Reply:
I agree with @tgamblin on this

Reply:
fixed in 542bab6